src/foam/u2/view/EnumLegendView.js

```diff
@@ -0,0 +1,74 @@
+foam.CLASS({
+  package: 'foam.u2.view',
+  name: 'EnumLegendView',
+  extends: 'foam.u2.View',
+
+  css: `
+  ^container {
+    border-style: solid;
+    border-width: thin;
+  }
+  ^eachValue {
+    display: inline-flex;
+    padding: 12px;
+    text-align: justify;
+  }
+  ^badge {
```

Comment:
badge class is duplicated here. It may be better to use that `Style` model to keep the CSS is one place (we could try changing that to a CSS axiom later). In fact this class needs to be updated to add `^badge > * { background-color: none }` because the coloured rectangle on the inside is creating an artifact that doesn't look very clean.

Reply:
adjustments coming